app/models/item.rb

```diff
@@ -5,22 +5,47 @@ class Item < ActiveRecord::Base
   validates :text, :list_id, presence: true
 
   scope :incompleted, -> { where.not('done') }
+  scope :completed, -> { where('done') }
+  scope :for_today, -> {
+    where('completed_at >= ? AND completed_at <= ?',
+    Date.today.to_time.beginning_of_day,
+    Date.today.to_time.end_of_day)
+  }
```

Comment:
Please consider the following:
- Prefer `{...}` over `do...end` for single-line blocks. Avoid using `{...}` for multi-line blocks. See [the details](https://github.com/bbatsov/ruby-style-guide#user-content-single-line-blocks).
- Adjust the alignment of the parameters. See [the details](https://github.com/bbatsov/ruby-style-guide#no-double-indent).